jdi-light-parallel-examples/src/main/java/io/github/epam/entities/Users.java

```diff
@@ -0,0 +1,19 @@
+package io.github.epam.entities;
+
+public class Users {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Users' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)